front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/HeroBanner/index.tsx

```diff
@@ -38,8 +37,8 @@ const EditCustomPageHeroBannerForm = ({
 }: InjectedIntlProps) => {
   const localize = useLocalize();
   const [isLoading, setIsLoading] = useState(false);
-  const [hasCTAMultilocError, setHasCTAMultilocError] = useState(false);
-  const [hasCTAUrlError, setHasCTAUrlError] = useState(false);
+  const [apiErrors, setApiErrors] = useState<CLErrors | null>(null);
```

Comment:
Ideally via hook forms, but as we don't have them, this is the way to do apiErrors in the codebase. The logic to define API errors is explained [here](https://www.notion.so/citizenlab/Forms-3d721df532d040e1bd4e399470d0d8a1#107093c00dd043b6a12a4d8b9776133c).

The logic is in the Error component, so no need for us to manage this manually.